README.md

```diff
@@ -89,3 +89,25 @@ $ git tag -d v1.2.3                 # delete tag v1.2.3 locally
 $ git push origin v1.2.3            # push tag v1.2.3
 $ git push -d origin v1.2.3         # delete tag v1.2.3 remotely
 ```
+
+## Design patterns
+
+#### JS concepts
+
+- The repo is set up to import individual components, which then in turn import their own scss modules. 
+- Preference for hooks and functional components over classes.
+- Preference for proptypes validation over typescript and .d.ts files. 
+- Please try to avoid using `<Media>` and `<Spacer>` components for now, they will be deprecated soon. For media queries, import `@include` rules from the media scss file as described below.
+
+#### CSS/SCSS concepts
+
+- CSS variables are the core of styling in the EDS. 
+  - All CSS variables are available on all pages in both monorepo and cms; they are the single source of truth for brand colors, breakpoints, and other commonly reused bits of css.
+- The EDS is (to an extent) designed as a set of immutable components in terms of interior JS logic as well as styling. New styling of existing components should be handled by PR's on the EDS, not by writing local overrides in files. Due to the generated hashed classnames from scss modules, it is almost impossible to sidestep styling rules of the EDS repo from monorepo or cms.
```

Comment:
I'm fine with this, but less the last sentence:

> Due to the generated hashed classnames from scss modules, it is almost impossible to sidestep styling rules of the EDS repo from monorepo or cms.

Let's take that out. It's implementation detail and I would also argue from experience that it is in fact possible to still override (although of course we don't want folks to do so)